File: wood-backend/src/models/series.js
const mongoose = require('mongoose');

const seriesSchema = new mongoose.Schema({
  title: {
    type: String,
    required: true,
    index: true,
  },
  description: {
    type: String,
    required: true,
  },
  genre: {
    type: String,
    required: true,
    index: true,
  },
  coverImageUrl: {
    type: String,
    required: true,
  },
});

module.exports = mongoose.model('Series', seriesSchema);

File: flutter_wood/lib/models/series.dart
class Series {
  final String id;
  final String title;
  final String description;
  final String genre;
  final String coverImageUrl;

  Series({
    required this.id,
    required this.title,
    required this.description,
    required this.genre,
    required this.coverImageUrl,
  });

  factory Series.fromJson(Map<String, dynamic> json) {
    return Series(
      id: json['_id'],
      title: json['title'],
      description: json['description'],
      genre: json['genre'],
      coverImageUrl: json['coverImageUrl'],
    );
  }
}